client/src/components/dataset/sidebars/GenerateGeopoints.jsx

```diff
@@ -54,6 +55,27 @@ export default class GenerateGeopoints extends Component {
     this.handleChangeColumnTitleGeo = this.handleChangeColumnTitleGeo.bind(this);
   }
 
+  getColumns(latOrLong) {
```

Comment:
I guess this logic can be unit tested? it is possible?

Reply:
I'd have to extract it out to a separate function to unit test it. Or else I'd be testing the whole component together